addons/udes_stock/models/stock_move.py

```diff
@@ -451,31 +451,40 @@ def refactor_action_group_by_move_line_key(self):
                 group of stock.moves
             attach the stock.moves and stock.move.lines to the new picking.
         """
-        Move = self.env['stock.move']
-        Picking = self.env['stock.picking']
 
         picking_type = self.mapped('picking_type_id')
         picking_type.ensure_one()
 
         if not picking_type.u_move_line_key_format:
             return
 
-        pickings = self.mapped('picking_id')
         mls_by_key = self.mapped('move_line_ids').group_by_key()
 
+        return self.refactor_by_move_line_groups(mls_by_key.items())
```

Comment:
For consistency, can we do the same at `refactor_action_group_by_move_key` ?

Reply:
Sure :)